### PR TITLE
feat(overthebox.details): display service id into infos

### PR DESCRIPTION
**Pull request by Steffy29**
| Question         | Answer
| ---------------- | ---
| Branch?          | `develop`
| Bug fix?         | no
| New feature?     | yes
| Breaking change? | no
| Tickets          | Fix #UXCT-429
| License          | BSD 3-Clause

- [x] Try to keep pull requests small so they can be easily reviewed.
- [x] Commits are signed-off
- [x] Only FR translations have been updated
- [x] Branch is up-to-date with target branch
- [x] Lint has passed locally
- [x] Standalone app was ran and tested locally
- [x] Ticket reference is mentioned in linked commits (internal only)
- [ ] ~Breaking change is mentioned in relevant commits~

## Description

Into detail information, display service id with a label to be more clear for customer

### CODE REVIEW

File: packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html [hunk 1]
```diff
@@ -135,6 +135,21 @@
...
+                    <div class="oui-tile__item">
+                        <div class="oui-tile__definition">
+                            <strong
+                                class="oui-tile__term d-block"
+                                data-translate="overTheBox_serviceId"
+                            ></strong>
+                            <div class="oui-tile__description">
+                                <span
+                                    class="d-block mb-2"
+                                    data-ng-bind="$ctrl.serviceName"
```
Comment (antleblanc) [inline packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html:147]: **nitpick:** `span` element is useless. You can apply `data-ng-bind="$ctrl.serviceName"` directly on the parent and get rid of the `span` element.

**Review by antleblanc — CHANGES_REQUESTED** [review]

File: packages/manager/modules/overthebox/src/overthebox/details/translations/Messages_fr_FR.json [hunk 2]
```diff
@@ -84,5 +84,6 @@
   "overTheBox_last_seen_minutes_ago": "Il y a {{lastSeenDiff}} environ",
   "overTheBox_reverse_dns": "Reverse DNS",
   "overTheBox_detail_link_guide": "Guides",
-  "overthebox_detail_links": "Liens"
+  "overthebox_detail_links": "Liens",
+  "overTheBox_serviceId": "Identifiant du service :"
```
Comment (JeremyDec) [inline packages/manager/modules/overthebox/src/overthebox/details/translations/Messages_fr_FR.json:88]: **nitpick**: To keep consistency
```suggestion
  "overTheBox_service_id": "Identifiant du service :"
```

**Review by antleblanc — DISMISSED** [review]
Could you please address @JeremyDec's comment?
Thank you!

**Review by JeremyDec — DISMISSED** [review]

File: packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html [hunk 3]
```diff
@@ -367,10 +379,18 @@
...
-                            <strong class="oui-tile__term d-block" data-translate="overthebox_detail_links">
+                            <strong
+                                class="oui-tile__term d-block"
+                                data-translate="overthebox_detail_links"
+                            >
                             </strong>
-                            <oui-tile-button data-ng-href="{{:: $ctrl.guidesLink }}" external>
-                                <span data-translate="overTheBox_detail_link_guide"></span>
+                            <oui-tile-button
+                                data-ng-href="{{:: $ctrl.guidesLink }}"
```
Comment (marie-j) [inline packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html:388]: https://ovh.github.io/ovh-ui-kit/?path=/docs/design-system-components-tile-webcomponent--with-buttons
`oui-tile-button` supports `href` attribute

**Review by marie-j — CHANGES_REQUESTED** [review]

File: packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html (hunk 3 above)
Comment (Steffy29) [reply packages/manager/modules/overthebox/src/overthebox/details/overTheBox-details.html:388]: Sorry but I don't understand your feedback. I use `href` for `oui-tile-button`. Those lines doesn't change, there are just more far in the code...

**Review by marie-j — APPROVED** [review]

**Review by JeremyDec — APPROVED** [review]

**Review by antleblanc — APPROVED** [review]

**Pull request merged**